TYPO3.Neos/Classes/TYPO3/Neos/Controller/Module/Management/AssetController.php

```diff
@@ -132,9 +132,9 @@ public function deleteAction(\TYPO3\Media\Domain\Model\Asset $asset)
     {
         try {
             $this->assetService->getRepository($asset)->remove($asset);
-            $this->addFlashMessage('assetHasBeenDeleted', '', Message::SEVERITY_OK, [$asset->getLabel()], 1412375050);
+            $this->addFlashMessage('Asset "%s" has been deleted.', '', Message::SEVERITY_OK, [$asset->getLabel()], 1412375050);
         } catch (AssetServiceException $exception) {
-            $this->addFlashMessage('media.deleteRelatedNodes', '', Message::SEVERITY_WARNING, [], 1412422767);
+            $this->addFlashMessage('Asset could not be deleted, because there are still Nodes using it', '', Message::SEVERITY_WARNING, [], 1412422767);
```

Comment:
but again we should probably just put it in the media package for now (add the missing 1462308179) and then clean it up when we move everything to the `Neos.MediaBrowser` in #159